### PR TITLE
Replace abandoned Sensiolabs security checker

### PR DESCRIPTION
**Pull request by paras-malhotra**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | yes/no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

This PR replaces the abandoned [Sensiolabs security checker](https://github.com/sensiolabs/security-checker) with the [Enlightn security checker](https://github.com/enlightn/security-checker).

### CODE REVIEW

**Comment by paras-malhotra** [comment]
@Ocramius would be great if you could spare some time to review this PR. Since Sensiolabs security checker and API stopped working on Monday, Laminas Diagnostics users are without security checkers right now.

File: test/TestAsset/Check/SecurityAdvisory.php [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
+use Enlightn\SecurityChecker\AdvisoryAnalyzer;
 
 class SecurityAdvisory extends BaseCheck
 {
     /**
-     * @param SecurityChecker $securityChecker
+     * @param \Enlightn\SecurityChecker\AdvisoryAnalyzer $advisoryAnalyzer
      */
-    public function setSecurityChecker(SecurityChecker $securityChecker)
+    public function setAdvisoryAnalyzer(AdvisoryAnalyzer $advisoryAnalyzer)
```
Comment (Ocramius) [inline test/TestAsset/Check/SecurityAdvisory.php:19]: Overall LGTM, but signature changes require a major version bump :-\

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: test/TestAsset/Check/SecurityAdvisory.php (hunk 1 above)
Comment (paras-malhotra) [reply test/TestAsset/Check/SecurityAdvisory.php:19]: Hmm, I see. I guess there's no way around it given the Sensiolabs SecurityChecker type hinted class. Does that mean this PR needs to wait?

Comment (Ocramius) [reply test/TestAsset/Check/SecurityAdvisory.php:19]: We can certainly also release a new major, if needed

Comment (Ocramius) [reply test/TestAsset/Check/SecurityAdvisory.php:19]: @weierophinney probably need your input here

Comment (kbond) [reply test/TestAsset/Check/SecurityAdvisory.php:19]: Could an exception be made seeing as the SensioLabs SecurityChecker no longer works?

Comment (arueckauer) [reply test/TestAsset/Check/SecurityAdvisory.php:19]: The service itself issues a warning when invoked, don't see a need for throwing an exception.
> WARNING Security Advisory: The web service failed for an unknown reason (HTTP 403).

**Comment by paras-malhotra** [comment]
@Ocramius, can we merge this in and possibly push out a release? Would be happy to make any changes, if needed.

**Comment by Barbarians** [comment]
When will you validate this PR ?

File: test/TestAsset/Check/SecurityAdvisory.php (hunk 1 above)
Comment (Ocramius) [reply test/TestAsset/Check/SecurityAdvisory.php:19]: I misread this: this is a test asset, so no major release needed.

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
@Barbarians thanks for the poke: I indeed don't get/have enough time to check my github backlog.

**Pull request merged**